library/src/schemas/bigint/bigintAsync.ts

```diff
@@ -42,28 +46,13 @@ export function bigintAsync(
   arg2?: PipeAsync<bigint>
 ): BigintSchemaAsync {
   // Get error and pipe argument
-  const [error, pipe] = getDefaultArgs(arg1, arg2);
+  const [error, pipe = []] = getDefaultArgs(arg1, arg2);
```

Comment:
Maybe we should also allow `undefined` for `pipe` to avoid creating an object unnecessarily.

Reply:
Also I would prefer to outsource this to `getDefaultArgs` if we keep that, instead of adding it everywhere.

Reply:
Either/or, I don't think it matters too much. My personal preference is to reduce the number of existence checks.

Reply:
And I'm open to moving it to the function itself. When I was writing all of this, I ran into type errors trying to modify `getDefaultArgs` so this was the least friction for me at the time.

Reply:
Then I recommend to remove it completely and allow `undefined`. I think this is the best way to proceed for now. It might also make sense to add `TPipe` to the generics, so that the `.pipe` property is 1 to 1 the same as the `pipe` argument. However, we could also look at the generic thing in a future PR to reduce this on to the essentials.

Reply:
I'll take a look at implementing this generic tonight, though I agree it would be a good follow-up PR.